webdriver_wharf/env.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+def str2bool(o):
+    if isinstance(o, basestring):
+        o = o.lower() in ['true', '1']
+    return o
+
+# Name of the docker image to use
+image_name = os.environ.get('WEBDRIVER_WHARF_IMAGE', 'cfmeqe/sel_ff_chrome')
+
+# Set to True if the container runs a VNC service
+vnc_enabled = str2bool(os.environ.get('WEBDRIVER_WHARF_IMAGE_VNC_ENABLED',
+    True))
+
+# Set to True if the container runs an http service
+http_enabled = str2bool(os.environ.get('WEBDRIVER_WHARF_IMAGE_HTTP_ENABLED',
+    True))
```

Comment:
These don't need to wrap. The project style is set to 100 column line length in setup.cfg (use flake8, not pep8)